crates/frontend/src/components/context_card.rs

```diff
@@ -0,0 +1,80 @@
+use leptos::*;
+use serde_json::{Map, Value};
+
+use crate::{
+    components::{
+        condition_pills::{types::Condition, ConditionPills},
+        table::{types::Column, Table},
+    },
+    types::Context,
+};
+
+#[component]
+pub fn context_card(
+    context: Context,
+    overrides: Vec<(String, Value)>,
```

Comment:
Better to take a map, no?

Reply:
Since all override managing functions return `Map<String, Value>` the onus is on the user to convert to vector and pass it to this function